### DISCUSSION TITLE
Добавить возможность внести еще одного пользователя в вебинтерфейс

### DISCUSSION THREAD

**Discussion by alexinc-2000** [mikopbx/Core#197] [Ideas (Идеи)]
Хочется видеть возможность добавления еще одного пользователя в веб-интерфейс управления mikopbx. В систему по ssh его добавлять необязательно.
Это даст возможность управлять системой нескольким людям, не передавая пароль. И отражать это в логах. Понимаю, что ACL делать долго и сложно, но на первых порах и не обязательно. Если есть возможность, то просто реализовать admin и user уровень.

**Comment by boffart** [comment]
https://github.com/mikopbx/Core/discussions/73
Есть схожее обсуждение. 
Да, в планах есть реализация такого функционала.

**Comment by bambr47** [comment]
+1 За данный функционал с возможностью ограничений доступа. Примеров масса, например начальнику отдела требуется прослушать разговоры подчиненных, но при этом не слушать разговоры админов, директоров и тд.

**Comment by jorikfon** [comment]
Сколько может стоить подобный модуль по вашему, например если это разовая оплата? :)

**Reply by bambr47** [reply]
если под одно конкретного клиента 50-80, если как доп для всех, 10-30 ИМХО.

**Reply by jorikfon** [reply]
Ок, делаем подобный модуль, но пока есть проблема в том как отображать матрицу доступа, очень много галочек получается, например для списка сотрудников нужно просмотр, редактирование, сохранение настроек, получение статусов и детальной отладочной информации, как минимум 5-6 галок на 1 пункт меню получается, сложно такое настраивать, но что-нибудь придумаем.

**Reply by bambr47** [reply]
Лично у меня основная задача выдать людям прослушку, но что бы они не могли слушать ничего кроме разрешенных.

**Reply by jorikfon** [reply]
@bambr47 слушать только определенные внутренние номера?

**Reply by bambr47** [reply]
Да, сценарий таков:
Начальник отдела продаж может слушать продажников
Директор может слушать всех
Руководитель техподдержки может слушать только своих покемонов =)

**Reply by jorikfon** [reply]
А 1С вы не используете? Там это намного проще реализовать.

**Reply by bambr47** [reply]
Нет, для этого 1с не надо, я стараюсь делать сервисы независимыми.

**Reply by EDautoAdmin** [reply]
Уважаемые коллеги. Для тех, кто не в теме. Мико продаёт модуль к 1с. Там всё есть. Они не будут навешивать 1Совский функционал на вебку АТСки для конкуренции со своим же решением. Покупайте модули 1С. Всё просто. :) :) :)

**Reply by napaster** [reply]
Не все используют для этого 1с. У кого то 1с идет только для бухгалтерии и находится вообще в облаке. Так что не всем подходит данное решение. А есть те кто принципиально не хочет это грузить в 1с. Там своего бардака хватает.

**Reply by jorikfon** [reply]
Модуль доступен для тестирования, установить можно в DEV версии MikoPBX https://docs.mikopbx.com/mikopbx/modules/miko/module-users-u-i
